backend/webapi/Data/PidpDbContext.cs

```diff
@@ -49,10 +49,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
     private void ApplyAudits()
     {
-        this.ChangeTracker.DetectChanges();
```

Comment:
`Entries()` already calls `DetectChanges()`, so we are calling it twice here. This line would be necessary if we set `Context.AutoDetectChanges = false`, but we don't.